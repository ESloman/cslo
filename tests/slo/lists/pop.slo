var x = [1, 2, 3];
println(x);

var z = x.pop();  # 3
println(x);
println(z);

var a = x.pop();  # 2
println(a);
println(x);

println(x.pop());  # 1
println(x);

println(x.pop());
println(x);
