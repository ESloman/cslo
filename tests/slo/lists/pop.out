list[3]: [1, 2, 3]
list[2]: [1, 2]
3
2
list[1]: [1]
1
list[0]: []
nil
list[0]: []
